pkg/airgap/update.go

```diff
@@ -96,6 +98,18 @@ func UpdateAppFromPath(a *apptypes.App, airgapRoot string, airgapBundlePath stri
 		return errors.Wrap(err, "failed to parse license from file")
 	}
 
+	missingPrereqs, err := GetMissingRequiredVersions(a, airgap)
+	if err != nil {
+		return errors.Wrapf(err, "failed to check required versions")
+	}
+
+	if len(missingPrereqs) > 0 {
+		return util.ActionableError{
+			NoRetry: true,
+			Message: fmt.Sprintf("This airgap bundle requires the following releases to be installed first: %s", strings.Join(missingPrereqs, ", ")),
```

Comment:
@ajp-io any suggestion on the wording here? the one we're using for online installs doesn't necessarily work here.

Reply:
@divolgin after discussing with @ajp-io:
```
This airgap bundle cannot be deployed because versions %s are required and must be uploaded first.
```